zetaclient/tss/tss_signer.go

```diff
@@ -74,6 +74,7 @@ type TSS struct {
 	logger        zerolog.Logger
 	Signers       []string
 	CoreBridge    interfaces.ZetaCoreBridger
+	KeySignMan    *KeySignManager
```

Comment:
```suggestion
	KeySignManager    *KeySignManager
```
I commented to rename this type in any case, but more generally, let's keep fully explicit names for field names